### DISCUSSION TITLE
description vs. message

### DISCUSSION THREAD

**Discussion by kuchta** [jj-vcs/jj#3655] [General]
If jj calls commit message description, shoudn't the parameter by named `-d` instead of `-m`?

**Comment by martinvonz** [comment]
See my reasoning in https://github.com/martinvonz/jj/commit/bad23cda74cdb8f43f9da7e4080fbb706b9f9209. That doesn't mean we can't change it, of course.

**Reply by kuchta** [reply]
🙂 I absolutely get it. Maybe description isn't the best term. What about -c --comment? It may be even better than description...

**Reply by martinvonz** [reply]
And rename `jj describe` to `jj comment`? Not saying we can't do that either, but I personally prefer `jj describe`. Let's hear what others think. If few people comment here, I'm going to assume it means they're okay with the status quo even though it's inconsistent.

**Comment by joyously** [comment]
> If jj calls commit message description

Where?
The action is describe and an optional parameter is the message. This makes sense, especially since it's the same as most VCS that came before.
If description is used in help text or docs, maybe that should be changed to message. (It reminds me of using both revision and commit.)

**Comment by kuchta** [comment]
@joyously If you want to combine different nomenclature from different systems, you inevitably end up with quite inconsistent dictionary like git, which I hope it's damage on sw industry this project tries to fix. It doesn't matter if we are talking about special command that does just that or parameter of another command that does that as a secondary operation of it's primary one....
